### ISSUE TITLE
Error running check disk

### ISSUE THREAD

**Issue by Izorkin** [DataDog/datadog-agent#2822]
**Output of the info page (if this is a bug)**
```
Getting the status from the agent.

==============
Agent (v6.8.2)
==============

  Status date: 2018-12-24 13:53:30.915510 MSK
  Pid: 22737
  Python Version: 2.7.15
  Logs:
  Check Runners: 4
  Log Level: info

  Paths
  =====
    Config File: /etc/datadog-agent/datadog.yaml
    conf.d: /etc/datadog-agent/conf.d
    checks.d: /etc/datadog-agent/checks.d

  Clocks
  ======
    NTP offset: 172µs
    System UTC time: 2018-12-24 13:53:30.915510 MSK

  Host Info
  =========
    bootTime: 2018-12-24 13:03:45.000000 MSK
    kernelVersion: 4.19.12
    os: linux
    platform: nixos
    platformVersion: &#34;19.03.git.0e19695 (Koi)&#34;
    procs: 599
    uptime: 47m2s
    virtualizationRole: host
    virtualizationSystem: kvm

  Hostnames
  =========
    hostname: ...
    socket-fqdn: ...
    socket-hostname: ...
    hostname provider: os
    unused hostname providers:
      aws: not retrieving hostname from AWS: the host is not an ECS instance, and other providers already retrieve non-default hostnames
      configuration/environment: hostname is empty
      fqdn: fork/exec /bin/hostname: no such file or directory
      gce: unable to retrieve hostname from GCE: Get http://169.254.169.254/computeMetadata/v1/instance/hostname: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)

=========
Collector
=========

  Running Checks
  ==============

    cpu
    ---
      Instance ID: cpu [OK]
      Total Runs: 10
      Metric Samples: Last Run: 6, Total: 54
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 0s


    disk (1.3.0)
    ------------
      Instance ID: disk:791ba3f653790e13 [ERROR]
      Total Runs: 11
      Metric Samples: Last Run: 216, Total: 2,376
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 97ms
      Error: not sure how to interpret line '   1       0 ram0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n'
      Traceback (most recent call last):
        File "/nix/store/xpk3p0p22rwdwckibhb7pgqgm820151s-python-2.7.15-env/lib/python2.7/site-packages/datadog_checks/checks/base.py", line 352, in run
          self.check(copy.deepcopy(self.instances[0]))
        File "/nix/store/xpk3p0p22rwdwckibhb7pgqgm820151s-python-2.7.15-env/lib/python2.7/site-packages/datadog_checks/disk/disk.py", line 43, in check
          self.collect_metrics_psutil()
        File "/nix/store/xpk3p0p22rwdwckibhb7pgqgm820151s-python-2.7.15-env/lib/python2.7/site-packages/datadog_checks/disk/disk.py", line 148, in collect_metrics_psutil
          self.collect_latency_metrics()
        File "/nix/store/xpk3p0p22rwdwckibhb7pgqgm820151s-python-2.7.15-env/lib/python2.7/site-packages/datadog_checks/disk/disk.py", line 230, in collect_latency_metrics
          for disk_name, disk in psutil.disk_io_counters(True).iteritems():
        File "/nix/store/xpk3p0p22rwdwckibhb7pgqgm820151s-python-2.7.15-env/lib/python2.7/site-packages/psutil/__init__.py", line 2026, in disk_io_counters
          rawdict = _psplatform.disk_io_counters(**kwargs)
        File "/nix/store/xpk3p0p22rwdwckibhb7pgqgm820151s-python-2.7.15-env/lib/python2.7/site-packages/psutil/_pslinux.py", line 1115, in disk_io_counters
          for entry in gen:
        File "/nix/store/xpk3p0p22rwdwckibhb7pgqgm820151s-python-2.7.15-env/lib/python2.7/site-packages/psutil/_pslinux.py", line 1088, in read_procfs
          raise ValueError("not sure how to interpret line %r" % line)
      ValueError: not sure how to interpret line '   1       0 ram0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n'

    file_handle
    -----------
      Instance ID: file_handle [OK]
      Total Runs: 11
      Metric Samples: Last Run: 5, Total: 55
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 0s


    io
    --
      Instance ID: io [OK]
      Total Runs: 11
      Metric Samples: Last Run: 338, Total: 3,484
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 0s


    load
    ----
      Instance ID: load [OK]
      Total Runs: 11
      Metric Samples: Last Run: 6, Total: 66
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 0s


    memory
    ------
      Instance ID: memory [OK]
      Total Runs: 11
      Metric Samples: Last Run: 17, Total: 187
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 0s


    network (1.6.1)
    ---------------
      Instance ID: network:e9bd34fadc0b2888 [OK]
      Total Runs: 11
      Metric Samples: Last Run: 56, Total: 616
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 1ms


    ntp
    ---
      Instance ID: ntp:b4579e02d1981c12 [OK]
      Total Runs: 11
      Metric Samples: Last Run: 1, Total: 11
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 1, Total: 11
      Average Execution Time : 5ms


    uptime
    ------
      Instance ID: uptime [OK]
      Total Runs: 11
      Metric Samples: Last Run: 1, Total: 11
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 0s

  Loading Errors
  ==============
    apm
    ---
      Core Check Loader:
        Check apm not found in Catalog

      Python Check Loader:
        No module named apm

    winproc
    -------
      Core Check Loader:
        Check winproc not found in Catalog

      Python Check Loader:
        No module named winproc

========
JMXFetch
========

  Initialized checks
  ==================
    no checks

  Failed checks
  =============
    no checks

=========
Forwarder
=========

  Transactions
  ============
    CheckRunsV1: 11
    Dropped: 0
    DroppedOnInput: 0
    Events: 0
    HostMetadata: 0
    IntakeV1: 2
    Metadata: 0
    Requeued: 0
    Retried: 0
    RetryQueueSize: 0
    Series: 0
    ServiceChecks: 0
    SketchSeries: 0
    Success: 22
    TimeseriesV1: 11

  API Keys status
  ===============
    API key ending with ... on endpoint https://app.datadoghq.com: API Key valid

==========
Logs Agent
==========

  Logs Agent is not running

=========
Aggregator
=========
  Checks Metric Sample: 7,035
  Event: 1
  Events Flushed: 1
  Number Of Flushes: 11
  Series Flushed: 6,184
  Service Check: 99
  Service Checks Flushed: 110

=========
DogStatsD
=========
  Event Packets: 0
  Event Parse Errors: 0
  Metric Packets: 0
  Metric Parse Errors: 0
  Service Check Packets: 0
  Service Check Parse Errors: 0

```

Configuration
 datadog.yaml:
```
{"additional_checksd":"/etc/datadog-agent/checks.d","confd_path":"/etc/datadog-agent/conf.d","dd_url":"https://app.datadoghq.com","skip_ssl_validation":false,"tags":"home, server","use_dogstatsd":true}
```
conf.d/disk.d/conf.yaml:
```
{"init_config":{},"instances":[{"use-mount":"false"}]}
```

**Comment by Izorkin** [comment]
This error fixed in psutil https://github.com/giampaolo/psutil/commit/8f99f3782663959062ee868bbfdbc336307a3a4d

**Comment by rmoriz** [comment]
This is really annoying :(

**Comment by Izorkin** [comment]
@rmoriz you need install psutil from sources - https://github.com/giampaolo/psutil/blob/master/INSTALL.rst#install-from-sources

**Comment by rmoriz** [comment]
I‘m currently patching the source of psutil that ships with datadog-agent using Chef on all machines 😞

**Comment by lukyanetsv** [comment]
Workaround for me - 
```
FROM datadog/agent:6.9.0-jmx

# update psutil to 5.5.0
# https://github.com/giampaolo/psutil/pull/1360
RUN apt-get update \
 && apt-get install --no-install-recommends -y build-essential \
 && /opt/datadog-agent/embedded/bin/pip install psutil==5.5.0 \
 && apt-get purge -y build-essential \
 && apt-get autoremove -y \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

**Comment by neverping** [comment]
In case you came to this issue using Google, this issue was fixed in Datadog 6.10.x and newer for Agents 6.x.
